naughty/fedora-35/2787-selinux-ioctl-iptables-containers

```diff
@@ -0,0 +1 @@
+grep -E 'avc:  denied  { ioctl } for * comm="iptables" path="/var/lib/containers/storage/*" dev="overlay" * scontext=unconfined_u:system_r:iptables_t:* tcontext=system_u:object_r:container_file_t:s0:*'
```

Comment:
so.. what's that grep here? that certainly doesn't belong here, and it wouldn't match either